docs/source/learn/core_notebooks/pymc_aesara.ipynb

```diff
@@ -0,0 +1,2039 @@
+{
```

Comment:
<p>The image is not rendering correctly: https://pymcio--5721.org.readthedocs.build/projects/docs/en/5721/learn/core_notebooks/pymc_aesara.html#introduction-to-aesara. Not sure how you added it to the notebook. If you aren't, I'd suggest using</p><p> </p><pre class="ql-syntax" spellcheck="false">![aesara logo](https://raw.githubusercontent.com/aesara-devs/aesara/main/doc/images/aesara_logo_2400.png)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Thanks! These images came from the original notebook. Now that I know how to preview the notebook I will take care of the images 👍

Reply:
<p>+1</p>